package.json

```
@@ -14,6 +14,7 @@
     "deploy-matic": "graph deploy snapshot-labs/snapshot-polygon subgraph.matic.yaml --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/",
     "deploy-bsc": "graph deploy snapshot-labs/snapshot-binance-smart-chain subgraph.bsc.yaml --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/",
     "deploy-gnosis-chain": "graph deploy snapshot-labs/snapshot-gnosis-chain subgraph.gnosis-chain.yaml --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/",
+    "arbitrum-one": "graph deploy snapshot-labs/snapshot-arbitrum-one subgraph.arbitrum-one.yaml --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/",
```

Can you change everywhere from arbitrum-one to arbitrum?
```suggestion
    "deploy-arbitrum": "graph deploy snapshot-labs/snapshot-arbitrum subgraph.arbitrum.yaml --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/",
```

---

@bonustrack Done, changed it everywhere except, ``` network: arbitrum-one``` because of this https://thegraph.com/docs/en/deploying/deploying-a-subgraph-to-hosted/